sycl/include/CL/sycl/program.hpp

```diff
@@ -322,7 +322,17 @@ class __SYCL_EXPORT program {
 #endif // __SYCL_DEVICE_ONLY__
   }
 
+  /// Gets the native handle of the SYCL platform.
+  ///
+  /// \return a native handle, the type of which defined by the backend.
+  template <backend BackendName>
+  auto get_native() const -> typename interop<BackendName, program>::type {
```

Comment:
CUDA's backend header doesn't have specialization for `program`

@Ruyk FYI

Reply:
Fixed, thx.